Hymns/Tags/TagStore.swift

```diff
@@ -9,6 +9,7 @@ protocol TagStore {
     func observeTagStatus(hymnIdentifier: HymnIdentifier, action: @escaping (Bool) -> Void) -> Notification
     func querySelectedTags(tagSelected: String?) -> Results<TagEntity>
     func queryTagsForHymn(hymnIdentifier: HymnIdentifier) -> Results<TagEntity>
+    func queryUniqueTags() -> [TagEntity]
```

Comment:
this is just my personal preference, but I prefer "get*" over "query*". So "getAllTags" or "getTagsForHymn". but up to you